apps/teca_deeplabv3p_ar_detect.in

```diff
@@ -26,8 +26,8 @@ def main():
 
     parser.add_argument(
         '--output_file', type=str, required=False,
-        default=r'pytorch_deeplabv3p_ar_detect_%t%.nc',
-        help=r'file pattern for output netcdf files (%t% is the time index)'
+        default=r'deeplabv3p_ar_detect_%%t%%.nc',
```

Comment:
% in the default value needs to be left alone, in the help string it needs to be escaped. the way to escape it is with %.